mis_builder/models/mis_report_instance.py

```diff
@@ -190,16 +203,18 @@ def _compute_dates(self):
             ("d", _("Day")),
             ("w", _("Week")),
             ("m", _("Month")),
+            ("q", _("quarterly")),
```

Comment:
To be in line with the others, this needs to be capitalized and the "ly" removed

Reply:
Capitalized